editor/src/clj/editor/input.clj

```diff
@@ -56,6 +58,20 @@
                        :shift (.isShiftDown scroll-event)
                        :meta (.isMetaDown scroll-event)
                        :control (.isControlDown scroll-event)))
+      :drag-over (let [drag-event ^DragEvent jfx-event]
+                   (.acceptTransferModes drag-event TransferMode/ANY)
+                   (assoc action
+                          :x (.getX drag-event)
+                          :y (.getY drag-event)))
+      :drag-dropped (let [drag-event ^DragEvent jfx-event]
+                      (.setDropCompleted drag-event true)
```

Comment:
Quoting javafx doc
> the gesture needs to be completed by calling setDropCompleted(Boolean) on the event. The Boolean argument indicates if the data has been transferred successfully or not. If it is not called, the gesture is considered unsuccessful.

Since this transforms the original jfx event to an action, we won't have access to the event from the input handler. The data has been transferred successfully is a way, but I think that's not exactly what the docs suggest.

Reply:
Hmm, maybe we could add the event to the action map so that we can use it if needed. As I understand, the effect of it is that if the drop is unsuccessful, there is an animation that moves the unsuccessfully-dragged icons back. Maybe we want that?